SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -1,136 +1,69 @@
 
-To connect a client-worker to a deployment from an offline state, you must use [`ClientTravel`](#aplayercontroller-clienttravel).
+By default in the editor and when [Spatial Networking]({{urlRoot}}/content/unreal-editor-interface/toolbars#switching-between-native-unreal-networking-and-spatialos-networking) is enabled clients and servers will automatically wait for a local SpatialOS deployment to be started by the editor before attempting to connect. If you wish to disable this functionality (e.g. if you want to start a deployment outside of the editor) you can change the "Auto-start local deployment" option in the [SpatialOS Editor Settings]({{urlRoot}}/content/unreal-editor-interface/editor-settings).
 
-The `LaunchClient.bat` (which we have provided) already includes the local host IP `127.0.0.1` which means client-workers launched this way will attempt to connect automatically using the [receptionist](#using-receptionist) flow.  
+#### With built clients
+By default, outside of PIE, clients do not connect to a SpatialOS deployment unless specific URL parameters are included as command-line arguments, not specifying a URL will result in an offline client. This is so you can implement your own connection flow, whether that be through an offline login screen, a connected lobby, etc. You can specify URL parameters at the command-line the same way as native Unreal with the first argument being the URL to use to travel to.
 
+To connect a client to a deployment from an offline state, you must use [`ClientTravel`](#aplayercontroller-clienttravel).
 
-#### With the Launcher
-When launching a client-worker from the SpatialOS Console using the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher), the client-worker will connect to SpatialOS by default. It has the `Locator` information required to connect to said deployment included as command-line arguments. When these `Locator` arguments are present, client-workers will attempt to connect automatically. Please note the launcher login tokens are only valid for 15 minutes.  
+The `LaunchClient.bat` helper script (which we have provided with the UnrealGDKExampleProject) already includes the local host IP `127.0.0.1` which means client launched this way will attempt to connect automatically using the [receptionist](#using-receptionist) flow.
 
-> Connecting by default when using the launcher is subject to change.
+#### With the SpatialOS Launcher
+When launching a client from the SpatialOS Console using the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher), the client will connect to the running SpatialOS cloud deployment by default. It has the `Locator` URL required to connect to said deployment included as command-line arguments. When these `Locator` arguments are present, client will attempt to connect automatically. Please note the launcher login tokens are only valid for 15 minutes.
```

Comment:
I think the launcher uses custom CLI arguments instead of a locator URL.

Reply:
Just checked some logs from a Launcher client and it's CLI was 
`locator.improbable.io//Game/Scavengers/Maps/Cascade/Cascade?locator?playeridentity=xxxxxxxxxxx?login=yyyyyyyyyyyyyyyy?Character=kali?TeamIndex=0?ClientTelemetrySessionId=bbbbbbbbbbbbbbbbbb?Name=Joshie?Ticket=aaaaaaaaaaaaaaaaaaa`